### ISSUE TITLE
Detailed settings for the linear evaluation

### ISSUE THREAD

**Issue by chenwydj** [google-research/simclr#5]
Could you brief the detailed settings for the linear evaluation? Thank you very much!
1. Learning rate and scheduler.
2. Batch size.
3. Epochs.
4. Optimizer: I saw you wrote L-BFGS in paper, but I did not find the keyword "lbfgs" or "bfgs" in your code.

Thanks a lot!

**Comment by chentingpc** [comment]
The linear evaluation hyper-parameters can be found in the ReadMe page.

PS, the linear evaluation for self-supervised learning is different from the linear evaluation for transfer learning experiments. The "lbfgs" optimizer was only used for transfer learning, NOT the self-supervised learning evaluation.

**Comment by chenwydj** [comment]
Thank you Ting!

- SGD with Nesterov momentum
- LR = 0.1
- no warmup
- cosine scheduler
- epochs = 100
- batch_size = 512

Is this correct for single-gpu Cifar10 linear evaluation? Thanks!

**Comment by chentingpc** [comment]
yes, this should work fine.

**Comment by chenwydj** [comment]
Thank you Ting!!

**Comment by deepankarvarma** [comment]
Can you share the notebook @chenwydj , thank you in advance.
